medical_patient_ethnicity/views/medical_patient_ethnicity_view.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+	<data>
+		<record id="medical_patient_ethnicity_view_tree" model="ir.ui.view">
+			<field name="name">medical.patient.ethnicity.tree</field>
+			<field name="model">medical.patient.ethnicity</field>
+			<field name="priority" eval="8" />
+			<field name="arch" type="xml">
+				<tree string="Patient Ethnicity">
+					<field name="notes" />
+					<field name="code" />
+					<field name="name" />
+				</tree>
+			</field>
+		</record>
+
+		<record id="medical_patient_ethnicity_view_form" model="ir.ui.view">
+			<field name="name">medical.patient.ethnicity.form</field>
+			<field name="model">medical.patient.ethnicity</field>
+			<field name="priority" eval="8" />
+			<field name="arch" type="xml">
+				<form string="Patient Ethnicity">
+					<group>
+						<field name="notes" />
+						<field name="code" />
+						<field name="name" />
+					</group>
+				</form>
+			</field>
+		</record>
+
+		<record id="medical_patient_ethnicity_view_search" model="ir.ui.view">
+			<field name="name">medical.patient.ethnicity.search</field>
+			<field name="model">medical.patient.ethnicity</field>
+			<field name="priority" eval="8" />
+			<field name="arch" type="xml">
+				<search string="Patient Ethnicity">
+					<field name="notes" />
+					<field name="code" />
+					<field name="name" />
+					<newline />
+					<group expand="0" string="Group By...">
```

Comment:
not necessary to group_by uniq values or text field. IMO the entierly group tag can be removed from the search view